discovery-infra/test_infra/tools/assets.py

```diff
@@ -0,0 +1,39 @@
+import json
+import logging
+from munch import Munch
+from test_infra import utils
+from test_infra import consts
+
+
+class Assets(object):
```

Comment:
assest represent a networking asset

Reply:
It still isn't clear to me. But let's at least add filedoc or docstring to the class

Reply:
I agree that the name Asset is unclear. But Lock is also not represent entirely this class. 
How about NetworkAsset? 
Ill also add a docstring, no problem.

Reply:
first of all it is not Lock for sure. I like "asset" name. Not sure what is not clear in it

Reply:
Because asset is too generic. It isn't clear what does it do or who owns it, what is it related to.
If you would ask me out-of-context what is an "asset" I would probably say either a token/credentials security related

Reply:
Assets meant to be generic, if u want to add specific asset type u can. Just inherit